dll/win32/dbghelp/symbol.c

```diff
@@ -109,7 +109,7 @@ DWORD             symt_ptr2index(struct module* module, const struct symt* sym)
 
 struct symt*      symt_index2ptr(struct module* module, DWORD id)
 {
-#ifdef __x86_64__
+#if defined(__x86_64__) || (defined(__arm64__) && defined(__APPLE__))
```

Comment:
I think this should be common for any arm64:
```suggestion
#if defined(__x86_64__) || (defined(__arm64__)
```